main.js

```diff
@@ -0,0 +1,12 @@
+const clicks = document.getElementById('clicks');
+
+window.onload = () => {
+    let counter = sessionStorage.getItem('counter') || 0; 
+    clicks.textContent = counter;    
+
+    window.onclick = () => {
+        counter++;
+        clicks.textContent = counter;
```

Comment:
You duplicated this row. Create functions for render counter and for increment counter